### PR TITLE
Add mock flag

### LINKED ISSUE #615: Default PDF Name if -o not provided
image

PDF file name should be: levelsio-05-Nov-2020-12-41PM.pdf

`node . -i 1324164916287332353 -o david1`

if -o david1 omitted default PDF file name should be generated.

### PR DESCRIPTION
**Pull request by johnjacobkenny**
## Description
Add the mock flag
Also add npm scripts for easy access
Closes #615 
## Attach Screenshot

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

File: twindle-cli/index.js [hunk 1]
```diff
@@ -9,19 +9,28 @@ const { sendToKindle } = require("./utils/send-to-kindle");
...
+    outputFilename,
+    tweetId,
+    kindleEmail,
+    mock,
+  } = getCommandlineArgs(process.argv);
 
   try {
-    const tweets = await getTweetsFromTweetId(tweetId);
+    // this next line is wrong
+    let tweets = require("../twindle-core/twitter_responses/response-version2-tweetthread.json");
```
Comment (johnjacobkenny) [inline twindle-cli/index.js:22]: @Mira-Alf Please guide me on the right file to use. This one is wrong.

File: twindle-cli/index.js [hunk 2]
```diff
@@ -9,19 +9,28 @@ const { sendToKindle } = require("./utils/send-to-kindle");
...
+    outputFilename,
+    tweetId,
+    kindleEmail,
+    mock,
+  } = getCommandlineArgs(process.argv);
 
   try {
-    const tweets = await getTweetsFromTweetId(tweetId);
+    // this next line is wrong
+    let tweets = require("./twitter/twitter_responses/response-version2-tweetthread.json");
```
Comment (johnjacobkenny) [inline twindle-cli/index.js:22]: @Mira-Alf please advise on the correctness of this code, which mock json to use.

File: twindle-cli/package.json [hunk 3]
```diff
@@ -1,37 +1,38 @@
...
-}
+{
+  "name": "twindle",
+  "version": "0.0.1",
+  "description": "This cli fetches a twitter thread and converts it into PDF, ePub, Mobi, etc",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start:mock": "node index.js -m",
+    "start:thread": "node index.js -i 1323618091511607296"
```
Comment (johnjacobkenny) [inline twindle-cli/package.json:9]: @proful does this look good

Comment (proful) [reply twindle-cli/package.json:9]: How the pdf file name looks?

**Comment by proful** [comment]
@johnjacobkenny I was thinking to mock Twitter api response. So that we can keep on working on Twitter modules as well.

**Comment by johnjacobkenny** [comment]
Yeah, more work needs to go into the mock section. Right now it is working as a mock over the entire twitter processing so that the renderer code can be tested.

**Pull request merged**